main.py

```diff
@@ -0,0 +1,53 @@
+import sys
+from PIL import Image
+from pathlib import Path
+from random import randint
+
+
+def getimages(location):
+    image = []
+    validimages = [".jpg", ".jpeg", ".png"]
+    p = Path(location)
+    try:
+        for x in p.iterdir():
+            ext = Path(x).suffix
+            if x.is_file() and ext in validimages:
+                image.append(x)
+    except FileNotFoundError as e:
+        print(e)
+        print("No such file exists")
+        sys.exit()
+    except Exception as e:
+        print(e)
+        print("There was a problem")
+        sys.exit()
+    else:
+        if len(image) != 0:
+            return image
+
+
+def showimage(img: list):
```

Comment:
shortened name that doesn't explain the variable properly. would you assume that a variable named img would be a list of images?
in general, the only times you would want to write a shortened name would be in case of iterators (i,j,k) or errors (e) and that is only if you don't have a better name for them.